tests/src/toCKB_lockscript/mod.rs

```diff
@@ -1 +1,192 @@
+use ckb_testtool::{builtin::ALWAYS_SUCCESS, context::Context};
+use ckb_tool::ckb_types::{
+    bytes::Bytes,
+    core::{TransactionBuilder, TransactionView},
+    packed::*,
+    prelude::*,
+};
+use ckb_tool::{ckb_error::assert_error_eq, ckb_script::ScriptError};
 
+use crate::Loader;
+
+const MAX_CYCLES: u64 = 10_000_000;
+
+#[derive(Clone, Copy)]
+struct CellCase {
+    is_type_toCKB_cell: bool,
+    is_valid_toCKB_cell: bool,
+}
+
+#[repr(i8)]
+pub enum Error {
+    IndexOutOfBound = 1,
+    ItemMissing,
+    LengthNotEnough,
+    Encoding,
+    // Add customized errors here...
+    InvalidToCKBCell,
+}
+
+fn get_lock_script_args(type_script: Script, is_ToCKBCell: bool) -> Bytes {
```

Comment:
no need

Reply:
It's used for a  test case which  there is no toCKB cell  which  `lockscript args`  are not equal to the `typescript hash` 。